doped/utils/stenciling.py

```
@@ -0,0 +1,1122 @@
+    apply_symm_op_to_struct,
+    get_clean_structure,
+    get_sga,
+    translate_structure,
+)
+
+
+def get_defect_in_supercell(
+    defect_entry: DefectEntry,
+    target_supercell: Structure,
+    check_bulk: bool = True,
+    target_frac_coords: np.ndarray[float] | list[float] | bool = True,
```

Pylint (3.3.7)

[error] Value 'np.ndarray' is unsubscriptable

(E1136)